gui/build.py

```diff
@@ -79,25 +81,20 @@ def do_bundle(options):
     for dirname in ('build', 'dist'):
         if os.path.isdir(dirname):
             shutil.rmtree(dirname)
-    # create a intaller
-    python_lib_path = os.path.join(options.python_path, "Lib")
-    makensis_path = os.path.join(options.nsis_path, "makensis.exe")
+    # create a installer
     gui_path = os.path.realpath('.')
     moz_path = os.path.realpath('..')
-    python_dir = os.path.join(
-        os.path.dirname(sys.executable),
-        "Scripts"
-    )
-    args = []
-    args.append('--icon=wininst/app_icon.ico')
-    args.append('--base-name=Win32GUI')
-    args.append('--include-path=%s;%s;%s\site-packages;%s'
-                % (gui_path, moz_path, python_lib_path, python_lib_path))
-    args.append('--target-name=mozregression-gui.exe')
-    args.append('--target-dir=dist')
-    args.append('mozregui/main.py')
-    call(sys.executable, os.path.join(python_dir, 'cxfreeze'), *args)
-    call(makensis_path, 'wininst.nsi', cwd='wininst')
+    if IS_WIN:
+        python_lib_path = os.path.join(options.python_path, "Lib")
```

Comment:
Hmm, I think we should set this in the default '--python-path':
default='C:\\Python27\\Lib'
This way we won't have to add Lib for windows. Thinking more about it, I think that we should rename the option to '--python-lib-path', because it is the lib path of python (note that for linux it is already the case: '/usr/lib/python2.7')
Also the variables in the code that are related should be renamed from python_path to python_lib_path.

Sorry for seeing this now, but I think it makes more sense. :)